File: week09/week09-3.py
#week09-3.py 學習計畫 Math 第2題
#LeetCode 1491. Average Salary Excluding the Minimum and Maximum Salary
class Solution:
    def average(self, salary: List[int]) -> float:
        salary.sort() #先小到大排好

        total = 0
        for i in range(1,len(salary)-1): #避開最左邊、最右邊
            total += salary[i]
        return total / (len(salary)-2) #除法，算出平均
        

File: week09/week09-3.cpp
///week09-3.cpp 學習計畫 Math 第2題

///LeetCode 1491. Average Salary Excluding the Minimum and Maximum Salary
class Solution {
public:
    double average(vector<int>& salary) {
        sort(salary.begin(),salary.end()); ///從小到大排好

        double total=0; ///陷阱，有小數點
        for(int i=1;i<salary.size()-1;i++){
            ///避開 最小salsry[0]、最大salary[N-1];
            total += salary[i];
        }
        return total / (salary.size()-2); ///平均，就是除法
    }
};